storage/revision.go

```diff
@@ -21,9 +21,15 @@ import "encoding/binary"
 // is a '_'. The last 8 bytes is the revision.sub in big-endian format.
 const revBytesLen = 8 + 1 + 8
 
+// revision is an index for key/value operations,
```

Comment:
A revision indicates a modify of the key-value space.
The set of changes that share same main revision changed the key-value space atomically.